### DISCUSSION TITLE
Wiki auto translator with chatgpt

### DISCUSSION THREAD

**Discussion by InsOpDe** [requarks/wiki#7215] [General]
Hi I build something:

https://github.com/team-parallax/wiki.js-translator

Limitations:
* Supports only markdown for now
* Links might be translated as well with the current prompt.
* Navigation is neither created nor linked

with this you can quickly translate your pages in an arbitrary language - have fun

**Comment by chatgptfrancais123** [comment]
You can use ChatGPT to complete your plan at [https://chatgpt-francais.com/](https://chatgpt-francais.com/)
